src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Utf8JsonWriterTests.cs

```diff
@@ -4422,7 +4421,7 @@ public void EscapeAsciiCharacters(JsonWriterOptions options)
 
         [Theory]
         [MemberData(nameof(JsonOptions_TestData))]
-        [OuterLoop("Too slow", typeof(PlatformDetection), nameof(PlatformDetection.IsBrowser))]
+        [OuterLoop("Too slow", typeof(PlatformDetection), nameof(PlatformDetection.IsMonoRuntime))]
```

Comment:
These could probably be restricted to IsMonoInterpreter not all of mono but this is probably fine